profiles/models/globalpermission.py

```diff
@@ -1,13 +1,74 @@
-from django.contrib.auth.models import User
+from django.contrib.auth.models import User, Permission
+from django.core.cache import cache
+from django.db.models import ManyToManyField
 from django.urls import reverse
+
 from profiles.models import AbstractScope
 
+default_user_permissions = [
+    # Lists
+    'service_catalog.list_instance',
+    'service_catalog.list_request',
+    'service_catalog.list_support',
+    'profiles.list_organization',
+    'profiles.list_team',
+    # Custom link
+    'service_catalog.view_customlink',
+    # Doc
+    'service_catalog.list_doc',
+    'service_catalog.view_doc',
+    # Portfolio
+    'service_catalog.list_portfolio',
+    'service_catalog.view_portfolio',
+    # Service
+    'service_catalog.list_service',
+    'service_catalog.view_service',
+    # Operation
+    'service_catalog.list_operation',
+    'service_catalog.view_operation',
+    'service_catalog.request_on_service',
+    # Request notification
+    'profiles.list_requestnotification',
+    'profiles.add_requestnotification',
+    'profiles.view_requestnotification',
+    'profiles.change_requestnotification',
+    'profiles.delete_requestnotification',
+    # Instance notification
+    'profiles.list_instancenotification',
+    'profiles.add_instancenotification',
+    'profiles.view_instancenotification',
+    'profiles.change_instancenotification',
+    'profiles.delete_instancenotification',
+]
+
```

Comment:
to be placed in migration

Reply:
or fixture

Reply:
moved in migration script

Reply:
must be moved in post migrate script (in apps.py) because Permission does not exist before